tools/tools.go

```diff
@@ -0,0 +1,26 @@
+// +build tools
+
+// To install the following tools at the version used by this repo run:
+// $ make tools
+// or
+// $ go generate -tags tools tools/tools.go
+
+package tools
+
+//go:generate go install github.com/hashicorp/go-bindata
+import _ "github.com/hashicorp/go-bindata"
+
+//go:generate go install github.com/golang/protobuf/proto
+import _ "github.com/golang/protobuf/proto"
+
+//go:generate go install github.com/golang/protobuf/protoc-gen-go
+import _ "github.com/golang/protobuf/protoc-gen-go"
+
+//go:generate go install github.com/grpc-ecosystem/grpc-gateway
+import _ "github.com/grpc-ecosystem/grpc-gateway"
+
+//go:generate go install github.com/mitchellh/protoc-gen-go-json
+import _ "github.com/mitchellh/protoc-gen-go-json"
+
+//go:generate go install github.com/vektra/mockery
+import _ "github.com/vektra/mockery/v2"
```

Comment:
Is there a good way to make sure we get mockery version `1.1.2`? Otherwise our server.proto arguments will fail since they use the single dash flags.

Reply:
To start I think you have to get rid of `/v2`

Reply:
That seemed to do it! Lol. Thought I tried that before and it didn't work but :shrug: Looks like that was it :sweat_smile: